src/test/java/software/amazon/awssdk/crt/test/HttpClientConnectionManagerTest.java

```diff
@@ -50,28 +51,33 @@ private HttpClientConnectionManager createConnectionManager(URI uri, int numThre
     }
 
     private HttpClientConnectionManager createConnectionManager(URI uri, int numThreads, int numConnections, String proxyHost, int proxyPort) {
-        try (EventLoopGroup eventLoopGroup = new EventLoopGroup(1);
-            HostResolver resolver = new HostResolver(eventLoopGroup);
-            ClientBootstrap bootstrap = new ClientBootstrap(eventLoopGroup, resolver);
-            SocketOptions sockOpts = new SocketOptions();
-            TlsContext tlsContext =  new TlsContext()) {
-
-            HttpProxyOptions proxyOptions = null;
-            if (proxyHost != null) {
-                proxyOptions = new HttpProxyOptions();
-                proxyOptions.setHost(proxyHost);
-                proxyOptions.setPort(proxyPort);
+        try (TlsContextOptions tlsOptions = TlsContextOptions.createDefaultClient()) {
```

Comment:
This (options + conditional ca add) gets done in a number of places.  Maybe create a util function that does it?

Reply:
Did it.